Scripts/RemediationScripts/Remediate-SetServiceBusMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,813 @@
+
+        .INPUTS
+        None. You cannot pipe objects to Set-SecureTLSVersionForServiceBusNamespaces.
+
+        .OUTPUTS
+        None. Set-SecureTLSVersionForServiceBusNamespaces does not return anything that can be piped and used as an input to another command.
+
+        .EXAMPLE
+        PS> Set-SecureTLSVersionForServiceBusNamespaces -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -DryRun
+
+        .EXAMPLE
+        PS> Set-SecureTLSVersionForServiceBusNamespaces -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck
+
+        .EXAMPLE
+        PS> Set-SecureTLSVersionForServiceBusNamespaces -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -InputFilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202209131040\SetSecureTLSVersionForServiceBusNamespaces\ServiceBusNamespacesInputFilePath.csv
+
+        .LINK
+        None
+    #>
+
+    param (
+        [String]
+        [Parameter(ParameterSetName = "DryRun", Mandatory = $true, HelpMessage = "Specifies the ID of the Subscription to be remediated")]
+        [Parameter(ParameterSetName = "WetRun", Mandatory = $true, HelpMessage = "Specifies the ID of the Subscription to be remediated")]
+        $SubscriptionId,
+
+        [Switch]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies a forceful remediation without any prompts")]
+        $Force,
+
+        [Switch]
+        [Parameter(ParameterSetName = "DryRun", HelpMessage = "Specifies validation of prerequisites for the command")]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies validation of prerequisites for the command")]
+        $PerformPreReqCheck,
+
+        [Switch]
+        [Parameter(ParameterSetName = "DryRun", Mandatory = $true, HelpMessage = "Specifies a dry run of the actual remediation")]
+        $DryRun,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies the path to the file to be used as input for the remediation")]
+        $InputFilePath,
+
+        [Switch]        
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies script is run as a subroutine of AutoRemediation Script")]
+        $AutoRemediation,
+
+        [String]
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies the path to the file to be used as input for the remediation when AutoRemediation switch is used")]
+        $Path,
+
+        [String]        
+        [Parameter(ParameterSetName = "WetRun", HelpMessage = "Specifies the time of creation of file to be used for logging remediation details when AutoRemediation switch is used")]
+        $TimeStamp
+    )
+
+    Write-Host $([Constants]::DoubleDashLine)
+
+    if ($PerformPreReqCheck) {
+        try {
+            Write-Host "[Step 1 of 4] Validating and installing the modules required to run the script and validating the user..."
```

Comment:
since this is a step, remove '...' from the end

Reply:
Done